src/driver/amdxdna/ve2_host_queue.h

```diff
@@ -0,0 +1,219 @@
+/* SPDX-License-Identifier: GPL-2.0 */
```

Comment:
The Handshake region is got updated in CERT. Could you please check once and update accordingly